The error is:

```
20:08:27 Undefined symbols for architecture x86_64:
20:08:27   "___sancov_lowest_stack", referenced from:
20:08:27       _LLVMFuzzerTestOneInput in fuzzer_test_simpler-a995e3.o
20:08:27       _$s4main12fuzzOneInput4Data4Sizes5Int32VSPys4Int8VG_SitF in fuzzer_test_simpler-a995e3.o
20:08:27       _$sSPys4Int8VGMa in fuzzer_test_simpler-a995e3.o
20:08:27       _$sSPys4Int8VGSPyxGs8_PointersWl in fuzzer_test_simpler-a995e3.o
20:08:27 ld: symbol(s) not found for architecture x86_64
```

It seems you have forgotten to guard one of the options on "is linux" condition.

--- a/lib/Frontend/CompilerInvocation.cpp
+++ b/lib/Frontend/CompilerInvocation.cpp
@@ -1130,9 +1130,14 @@ static bool ParseIRGenArgs(IRGenOptions &Opts, ArgList &Args,
 
     // Automatically set coverage flags, unless coverage type was explicitly
     // requested.
+    // Updated to match clang at Jul 2019.
     Opts.SanitizeCoverage.IndirectCalls = true;
     Opts.SanitizeCoverage.TraceCmp = true;
-    Opts.SanitizeCoverage.TracePCGuard = true;
+    Opts.SanitizeCoverage.PCTable = true;
+    if (Triple.isOSLinux()) {
+      Opts.SanitizeCoverage.StackDepth = true;
+    }
+    Opts.SanitizeCoverage.Inline8bitCounters = true;
     Opts.SanitizeCoverage.CoverageType = llvm::SanitizerCoverageOptions::SCK_Edge;
   }
 

--- a/test/IRGen/sanitize_coverage.swift
+++ b/test/IRGen/sanitize_coverage.swift
@@ -39,6 +39,6 @@ test()
 // FIXME: We need a way to distinguish the different types of coverage instrumentation
 // that isn't really fragile. For now just check there's at least one call to the function
 // used to increment coverage count at a particular PC.
-// SANCOV: call void @__sanitizer_cov
-
 // SANCOV_TRACE_CMP: call void @__sanitizer_cov_trace_cmp
+
+// SANCOV: call void @__sanitizer_cov

